cap-ci/cap-nightly-caasp.yaml

```diff
@@ -0,0 +1,67 @@
+workertags:
+  caasp4: suse-internal
+
+jobs:
+  deploy-k8s: false
+  deploy-kubecf: true
+  smoke-tests: true
+  cf-acceptance-tests-brain: true
+  sync-integration-tests: true
+  minibroker-integration-tests: true
+  cf-acceptance-tests: true
+  upgrade-kubecf: true
+  deploy-stratos: false
+  clean-cap: true
+  destroy-kubecf: false
```

Comment:
When you set `clean-cap`, it implies `destroy-kubecf`, right?

Reply:
No. `clean-cap` is for pet clusters to just delete cap install, not k8s. Therefore its set to `true` only in caasp CIs which uses pets.

Reply:
I don't think I follow the logic. The next field is named `destroy-kubecf` not `destroy-k8s`.

Reply:
yes. It's a naming convention issue. But it was not made part of acceptance criteria for this card to change that to destroy-k8s.

Reply:
Will create a new card right now to address this.

Reply:
You don't need to address everything in this PR! Still, it's very distressing that it's so poorly named and that it misled me during review. Could you create a card to address this, please?

Reply:
https://jira.suse.com/browse/CAP-1648